SConstruct

```diff
@@ -1080,13 +1080,14 @@ except ValueError:
 
 # Check for yaml-cpp library and checkout submodule if needed
 if env['system_yamlcpp'] in ('y', 'default'):
-    if conf.CheckCXXHeader('yaml-cpp/yaml.h', '""'):
+    # We need the Mark() function, which was added in version 0.5.3
+    if conf.CheckStatement('YAML::Node().Mark()', '#include "yaml-cpp/yaml.h"'):
         env['system_yamlcpp'] = True
         print("""INFO: Using system installation of yaml-cpp library.""")
 
     elif env['system_yamlcpp'] == 'y':
-        config_error('Expected system installation of yaml-cpp library, but it '
-            'could not be found.')
+        config_error("Expected system installation of yaml-cpp library, but it "
+            "could not be found or it is too old (0.6 or newer is required).")
```

Comment:
Should this say `0.5.3 or newer is required` considering the comment above?

Reply:
As discussed in #1029, our docs say that 0.6 is the minimum. I don't know how to check for 0.6.0 over 0.5.3, though, and I don't recall what specific issue motivated that choice.

Reply:
It may have been something to do with MSVC compatibility, IIRC. Anyhow, this looks fine to me then!